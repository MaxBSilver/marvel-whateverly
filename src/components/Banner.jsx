import React, { Component } from 'react'

export class Banner extends Component {
  render() {
    return (
      <div>
        
      </div>
    )
  }
}

export default Banner
