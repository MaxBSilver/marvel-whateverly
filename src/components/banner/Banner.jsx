import React from 'react'

function Banner() {
  return (
    <section className="banner-title">
      <h1>MARVELOUSLY</h1>
    </section>
  )
}

export default Banner
