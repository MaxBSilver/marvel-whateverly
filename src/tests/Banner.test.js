import React from 'react';
import Banner from '../components/banner/Banner';
import { shallow } from 'enzyme';

describe('Banner', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <Banner />
    )
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });

})