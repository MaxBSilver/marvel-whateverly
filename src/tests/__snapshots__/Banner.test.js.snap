// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner should match the snapshot with all the data passed in 1`] = `
<section
  className="banner-title"
>
  <h1>
    MARVELOUSLY
  </h1>
</section>
`;
